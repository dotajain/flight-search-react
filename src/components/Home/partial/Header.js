import React from 'react';

const Header = () => (
  <header className="header">
    <h1>Flight Search Engine</h1>
  </header>
);

export default Header;
