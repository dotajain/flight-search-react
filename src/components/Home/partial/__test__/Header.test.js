import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';

global.fetch = jest.fn(
  () =>
    new Promise(resolve => {
      process.nextTick(() => resolve({ json: () => ({}) }));
    }),
);

it('Properly rendered Header Component', () => {
  const component = renderer.create(<Header />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
