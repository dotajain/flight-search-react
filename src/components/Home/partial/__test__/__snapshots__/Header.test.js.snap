// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properly rendered Header Component 1`] = `
<header
  className="header"
>
  <h1>
    Flight Search Engine
  </h1>
</header>
`;
